tests/FSharp.Test.Utilities/ProjectGeneration.fs

```diff
@@ -678,11 +678,21 @@ module ProjectOperations =
         |> Option.map (fun s -> s.ToString())
         |> Option.defaultValue ""
 
+    let filterErrors (diagnostics: FSharpDiagnostic array) =
+        diagnostics
+        |> Array.filter (fun diag ->
+            match diag.Severity with
+            | FSharpDiagnosticSeverity.Hidden
+            | FSharpDiagnosticSeverity.Info
+            | FSharpDiagnosticSeverity.Warning -> false
+            | FSharpDiagnosticSeverity.Error -> true)
+
     let expectOk parseAndCheckResults _ =
```

Comment:
If you're in the mood, consider making error filtering a parameter for this function so that it can be more universal.